app/stylesheets/dashboard/charts/metric-status.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../config/colours';
+
+.icon {
```

Comment:
wrap all styles in `.metric-status {` (and add as a className to the div) so they are scoped to this component.